### ISSUE TITLE
Bring "Internet Search" button back again

### ISSUE THREAD

**Issue by letypequividelespoubelles** [osmandapp/OsmAnd#10463]
# 🚀 feature request

### Description
The internet search button has disappeared (since when ? ). As the local search is usually not very efficient, bring back the internet search would be profitable for everyone. 

Half the time, while searching for an address, the local search doesn't find it ... (I can give exemple below if you want).

Linked with issue #7716 and #8118 

Using Osmand 3.8.5 from F-Droid.

### Describe the solution you'd like
Bring "Internet Search" button back again

### Describe alternatives you've considered

Have a local search as powerfull as the internet search, but I guess it's difficult.

**Comment by letypequividelespoubelles** [comment]
If it's possible to have the internet button on top (#7716) when it'll be reintegrated, it would be wonderful.

Thanks !

**Comment by XandrexOSM** [comment]
hi, I have never seen that "internet search" button, but I believe that if I had it, I would not need another app to do searches first and send them to OSMand (in my case Acastus-Photon).

**Comment by scaidermern** [comment]
It would be a good idea to _not_ move this issue from milestone to milestone. The search function is one of OsmAnd's major drawback. It shouldn't be too hard to bring back the online search feature. Either via Nominatim or the photon search engine. After all, this feature has already been present a few versions ago.

**Comment by letypequividelespoubelles** [comment]
I use acastus photon (available on F-Droid) at the moment. I really hope I could soon use just OsmAnd ! :)

**Comment by vshcherb** [comment]
Sorry moving it back but Online Search is available in the list of Custom Searches, if you scroll down to the list

**Comment by scaidermern** [comment]
> Sorry moving it between sprints, that's a mistake. Online Search is available in the list of Custom Searches, if you scroll down to the list

Oh, indeed! This feature is well hidden.

In the past there was something like a "search online" button if the internal search couldn't find any results. This button no longer exists. I think it would be a good idea to bring it back. Otherwise most people will miss the online search functionality.

**Comment by vshcherb** [comment]
![telegram-cloud-photo-size-4-6037392217993230759-y](https://user-images.githubusercontent.com/1042025/167415078-41dedde9-9d7a-469b-8b2a-291b1902ffe5.jpg)
It still exists

**Comment by scaidermern** [comment]
Yes, the button exists. But only when going to search -> category -> online search. In this case you already know that Internet search is possible.

It doesn't appear when just going to the search menu. This will be what most users do. They don't know about the Internet search feature and the button won't appear.
